### ISSUE TITLE
pipelined stops responding and enforcement flow is not getting established when session is getting established

### ISSUE THREAD

**Issue by jpad-freedomfi** [magma/magma#7793]
Prior to submitting an issue, check to see if one has already been created. If there is currently an open issue, add a thumbs-up emoji to identify that it is also affecting you.

### Your Environment

- **Version:** 1.3.3 AGW, 1.5.1 orc8r
- **Affected Component:** Access Gateway
- **Affected Subcomponent:** pipelined
- **Deployment Environment:** e.g. AWS (orc8r), bare-metal (AGW)

### Describe the Issue

Customer site with 40 UEs, services have been up for ~2 weeks, today a backhaul outage caused eNB (CBRS) to drop the RF. When RF came back on, all UEs rushed to re-attach, during which pipelined got stuck. Remaining UEs that were not attached were unable to get enforcement flows installed when session is being created. This blocks internet access, causes CPE to flap. 

**To Reproduce**
1. Configure AGW in non-NAT mode
2. Repeat a loop of attach storms
3. Check sudo ovs-ofctl dump-flows gtp_br0 table=12
3. At some point observe that table=12 rules are not getting installed for some UEs. 
4. Observe that pipelined is hung

**Expected behavior**
pipelined should not hang

**Additional context**
Example logs of error IMSI with only drop rule vs. happy imsi with 3 rules
```
magma@agw:~$ sudo ovs-ofctl dump-flows gtp_br0 table=12 | grep 0x8ecde74cd2de9
 cookie=0x0, duration=24678.123s, table=12, n_packets=7590, n_bytes=546122, idle_age=1, priority=1,metadata=0x8ecde74cd2de9 actions=drop
magma@agw:~$ sudo ovs-ofctl dump-flows gtp_br0 table=12 | grep 0x8ecde74cd2e79
 cookie=0x13, duration=25011.159s, table=12, n_packets=2746754, n_bytes=291788480, idle_age=0, priority=65533,ip,reg1=0x1,metadata=0x8ecde74cd2e79,nw_src=10.200.81.116 actions=note:61.6c.6c.6f.77.6c.69.73.74.5f.73.69.64.2d.49.4d.53.49.33.31.34.30.33.30.30.30.30.30.32.32.39.39.31.2d.69.6e.74.65.72.6e.65.74.00.00.00.00,load:0x7->NXM_NX_PKT_MARK[],set_field:0x13->reg2,set_field:0x1b->reg4,resubmit(,13)
 cookie=0x13, duration=25011.160s, table=12, n_packets=5975619, n_bytes=10174489936, idle_age=0, priority=65533,ip,reg1=0x10,metadata=0x8ecde74cd2e79,nw_dst=10.200.81.116 actions=note:61.6c.6c.6f.77.6c.69.73.74.5f.73.69.64.2d.49.4d.53.49.33.31.34.30.33.30.30.30.30.30.32.32.39.39.31.2d.69.6e.74.65.72.6e.65.74.00.00.00.00,load:0x9->NXM_NX_PKT_MARK[],set_field:0x13->reg2,set_field:0x1b->reg4,resubmit(,13)
 cookie=0x0, duration=25011.154s, table=12, n_packets=54, n_bytes=3604, idle_age=65534, priority=1,metadata=0x8ecde74cd2e79 actions=drop
```

pipelined journalctl logs also stopped a long time ago, at the approximate time of the WAN outage and attach storm.
```
magma@agw:~$ sudo journalctl -fu magma@pipelined
-- Logs begin at Wed 2021-06-23 08:20:24 UTC. --
Jun 25 21:35:59 agw pipelined[19826]: RTNETLINK answers: No such file or directory
Jun 25 21:35:59 agw pipelined[19826]: RTNETLINK answers: No such file or directory
Jun 25 21:35:59 agw pipelined[19826]: We have an error talking to the kernel
Jun 25 21:35:59 agw pipelined[19826]: RTNETLINK answers: No such file or directory
Jun 25 21:35:59 agw pipelined[19826]: RTNETLINK answers: No such file or directory
Jun 25 21:35:59 agw pipelined[19826]: We have an error talking to the kernel
Jun 25 21:35:59 agw pipelined[19826]: RTNETLINK answers: No such file or directory
Jun 25 21:35:59 agw pipelined[19826]: RTNETLINK answers: No such file or directory
Jun 25 21:35:59 agw pipelined[19826]: We have an error talking to the kernel
Jun 25 21:35:59 agw pipelined[19826]: RTNETLINK answers: No such file or directory
^C
magma@agw:~$ date
Sat Jun 26 04:33:00 UTC 2021
```

**Comment by amarpad** [comment]
We need a SIGHUP handler that prints out the thread states otherwise debugging pipelined is like black magic. cc @koolzz @karthiksubraveti @themarwhal . Not sure where we landed on that discussion.

**Comment by themarwhal** [comment]
> We need a SIGHUP handler that prints out the thread states otherwise debugging pipelined is like black magic. cc @koolzz @karthiksubraveti @themarwhal . Not sure where we landed on that discussion.

That is implemented in 1.3: https://github.com/magma/magma/blob/v1.3/orc8r/gateway/python/magma/common/service.py#L337

**Comment by ekowtaylor** [comment]
@jpad-freedomfi this issue appears to have been fixed in v1.5

**Comment by ekowtaylor** [comment]
See related PR: https://github.com/magma/magma/pull/5240
